modules/retail/pages/retail-upgrade-branchserver.adoc

```diff
@@ -0,0 +1,7 @@
+[[retail-upgrade-branchserver]]
+= Upgrade {smr} Branch Server
+
+This section describes upgrading a {smr} Branch Server to the next SP (service pack).
+
+{smr} Branch Server is a client system such as a {productname} Proxy with additional {smr} features.
+For more information about upgrading a proxy client, see xref:upgrade:proxy-update.adoc[].
```

Comment:
A note that `Highstate should be applied on migrated branch/proxy to update retail functionality` should be here.
Just updating packages will not update retail configurations (i.e. new http mode will not be enabled by package update). Assuming that suma server is updated first.

Reply:
ok, thanks.  I hope I got it right.